server/server.go

```diff
@@ -1175,23 +1203,25 @@ func createRouter(eng *engine.Engine, logging, enableCors bool, corsHeaders stri
 			"/version":        getVersion,
 		},
 		"POST": {
-			"/auth":             postAuth,
-			"/attach":           postAttach,
-			"/container/create": postContainerCreate,
-			"/container/commit": postContainerCommit,
-			"/container/rename": postContainerRename,
-			"/exec":             postExec,
-			"/image/create":     postImageCreate,
-			"/image/build":      postImageBuild,
-			"/image/push":       postImagePush,
-			"/pod/create":       postPodCreate,
-			"/pod/labels":       postPodLabels,
-			"/pod/start":        postPodStart,
-			"/pod/stop":         postStop,
-			"/service/add":      postServiceAdd,
-			"/service/update":   postServiceUpdate,
-			"/tty/resize":       postTtyResize,
-			"/vm/create":        postVmCreate,
+			"/auth":              postAuth,
+			"/attach":            postAttach,
+			"/container/create":  postContainerCreate,
+			"/container/commit":  postContainerCommit,
+			"/container/rename":  postContainerRename,
+			"/container/pause":   postContainerPause,
+			"/container/unpause": postContainerUnpause,
```

Comment:
I think we should provide /pod/[un]pause, since we do it under the hood.
we may provide "/container/pause" in future when we can pause it individually